lib/chef/resource/chef_client_cron.rb

```diff
@@ -138,6 +138,12 @@ class ChefClientCron < Chef::Resource
         default: lazy { {} },
         description: "A Hash containing additional arbitrary environment variables under which the cron job will be run in the form of `({'ENV_VARIABLE' => 'VALUE'})`."
 
+      property :nice, [Integer, String],
+        description: "The process priority to run the #{Chef::Dist::CLIENT} process at. A value of -20 is the highest priority and 19 is the lowest priority.",
+        introduced: "16.5",
+        coerce: proc { |x| Integer(x) },
+        callbacks: { "should be a Integer between -20 and 19" => proc { |v| v >= -20 && v <= 19 } }
```

Comment:
```suggestion
        callbacks: { "must be an Integer between -20 and 19" => proc { |v| v >= -20 && v <= 19 } }
```

?

Reply:
We usually use should in the callbacks since that's the message presented on error.